frontend/packages/console-shared/src/types/resource.ts

```diff
@@ -53,3 +55,11 @@ export type OperatorBackedServiceKindMap = {
 };
 
 export type DeploymentStrategy = DEPLOYMENT_STRATEGY.recreate | DEPLOYMENT_STRATEGY.rolling;
+
+export type EventSourcesData = {
```

Comment:
if you use the utility `getDynamicEventSourcesModelRefs` I dont think you need to move the type here.